lib/ex_machina/ecto.ex

```diff
@@ -0,0 +1,58 @@
+defmodule ExMachina.Ecto do
+  defmacro __using__(opts) do
+    quote do
+      use ExMachina
+
+      @repo Dict.get(unquote(opts), :repo)
```

Comment:
Dict.fetch! would blow up if you try to call it without repo defined in the opts. However if you want something more friendly you could probably do something in save record like we did before. So you would check if `@repo` is defined. If it is, save it, if not raise a helpful error letting them know they need to pass `repo` in to the opts